src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2693,6 +2693,12 @@
 helps['bicep'] = """
 type: group
 short-summary: Bicep CLI command group.
+long-summary: |
+  Bicep CLI command group. There are two configurations that can be set for the command group, including bicep.check_version and bicep.use_binary_from_path.
+
+  bicep.check_version accepts Boolean values and defaults to True. If set to False, version checks for Bicep CLI installed via Azure CLI will be disabled.
+
+  The possible values of bicep.use_binary_from_path are if_found_in_ci and Booleans. The default value is if_found_in_ci.
```

Comment:
It is suggested to give an example of using `az config set`, which is more helpful for customers to understand.

Reply:
Such as:
https://github.com/Azure/azure-cli-extensions/blob/c2fa9aff05819e8b311a1614e787170e5ad98146/src/next/azext_next/_help.py#L14-L35

Reply:
Good suggestion. The same code is helpful. I've updated the help message to include examples.